chapters/6-analysis.tex

```diff
@@ -106,84 +105,82 @@ \subsection{Fitting construction into the data workflow}
 while forgetting to update the other.
 
 % Why construction is separate from analysis
-Construction of analysis data should be done right 
-data cleaning and before data analysis,
-according to the analysis plan.\index{Pre-analysis plan}
+Construction of analysis data should be done right
+after data cleaning and before data analysis,
+in accordance with the pre-analysis plan.\index{Pre-analysis plan}
 In practice, however, as you analyze the data,
-different constructed variables often become necessary,
-as well as different subsets of the sample
-and other transformations of the data.
-So you will need to adjust the analysis data accordingly.
-But even if construction and analysis are done concurrently,
+it is often useful to revisit construction,
+and explore different subsets of the same and other transformations of the data.
+Even if construction and analysis are done concurrently,
 you should always do the two in separate scripts.
 If every script that creates a table starts by loading a dataset,
 subsetting it, and manipulating variables,
 any edits to construction need to be replicated in all scripts.
-This increases the chances that at least one of them will have a different sample or variable definition.
+This increases the chances that at least one of them
+will have a different sample or variable definition.
 Doing all variable construction and data transformation
 in a unified script, separate from the analysis code, helps
 avoid this and ensures consistency across different outputs.
 
 \subsection{Integrating different data sources}
 
 % When merging is necessary and how to start thinking about it
-Combining or merging information from different data sources is often times necessary
-to create the analysis dataset.
+To create the analysis dataset,
+it is typically necessary to combine or merge information from different data sources.
 For example, you may merge administrative data with survey data
-to include demographic information in your analysis,
+in order to include demographic information in your analysis,
 or you may want to integrate geographic information
-in order to include location-specific controls.\sidenote{Such 
+in order to include location-specific controls.\sidenote{Such
 	operations are commonly called ``merges'' in Stata, and
-	``joins'' in R's \texttt{tidyverse} dialect. 
+	``joins'' in R's \texttt{tidyverse} dialect.
 	We will use the first term on the next few paragraphs.}
-To understand how to perform such operations, 
+To understand how to perform such operations,
 you will need to consider the unit of observation for each dataset,
 and their respective identifying variables.
 Merges are common sources of missing values,
 and can also result in observations being dropped.
 To avoid unintentional changes to your data,
 pay close attention to merge results,
-and add checks to your code verifying that they match what you expect.
+and add checks to your code to very that they match what you expect.
```

Comment:
Discussed in chapter 3 in relation to data flow charts.

Reply:
Done in 06f9055